WTWSMS/common/religions/00_religions.txt

```diff
@@ -1408,26 +1415,58 @@ muslim = {
 		uses_decadence = yes
 		uses_jizya_tax = yes
 	}
-	wahhabi = {
+	wahhabi = {  #this is the extremist sect practiced by ISIL, Al-Qaeda, and Saudi Arabia to this day, as such, it is extremely violent, as it remains totally unchanged from its inception in the seventh and eighth centuries
```

Comment:
@loup99 

I believe that to be incorrect. Wahhabis have always been ultra-conservative and extreme. They're basically the Puritans of Islam. I'm very proud of this addition; I consider it innovative, and I tried to balance the benefits the religion gives instead of just making them crazed. It gives players the chance to be total villains, and it fits in with Islam's history. It's a heresy I'd want to play!

From what I know about Wahhabism, it was a heresy that started several decades after Muhammed's death, in either the 700s AD or the 800s AD. Wahhabis saw the unusual tolerance practiced by the Mutalazites and other more "enlightened" Muslims and rejected any peace with the infidels. They holed up deep in Arabia and Mesopotamia and constantly existed as a thorn in the side of every Muslim empire for centuries on end. They have warred everything that didn't conform to them for centuries.

Once a minority, the Wahhabi sect of Islam is one of the most powerful, most influential, and most violent sects of Islam to ever exist (they only became powerful a few decades ago).

This is a very accurate description of the Wahhabi religious sect. Even though it appears as a heresy immediately, that doesn't mean it shouldn't be treated as it was known for when it actually spawned, two hundred years later.